api/bin/alembic_rebase_migrations.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python
+"""Rebase migrations and update Alembic conflict detection file upon a
+failing Git rebase off of master.
+
+If your feature branch contains new Alembic migrations and the master
+branch has more recent migrations, you'll have an error when rebasing
+off of master, something like:
+
+    Auto-merging api/alembic_version_conflict_detection.txt
+    CONFLICT (content): Merge conflict in api/alembic_version_conflict_detection.txt
+    error: could not apply 679ecbc01c...
+
+You must then update this file and the "down_revision" variable in the
+first "pre" and/or migration of your branch. This is cumbersome. This
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** 
```suggestion
first "pre" and/or "post" migration of your branch. This is cumbersome. This
‏